README.md

```diff
@@ -27,9 +27,13 @@ It is possible to run the operator in your local machine. This is useful for tes
     ```
     kubectl apply -f deploy/crds/k8s.nginx.org_nginxingresscontrollers_crd.yaml
     ```
+1. Apply the NGINX Ingress Controller CRDs:
+    ```
+    kubectl apply -f deploy/kic_crds
```

Comment:
https://github.com/nginxinc/nginx-ingress-operator/tree/add-kic-crds/deploy/kic_crds/ does not exist?

I think this was intended to be:
https://github.com/nginxinc/nginx-ingress-operator/tree/add-kic-crds/build/kic_crds/

```suggestion
    kubectl apply -f build/kic_crds
```

Reply:
Yess good catch @Dean-Coakley thanks